doc/reference/devices_gpu.md

```diff
@@ -51,8 +51,105 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
     lxc config device add <instance_name> <device_name> gpu gputype=physical pci=<pci_address>
 
+Add a specific GPU from the host system as a `physical` GPU device to an instance using the [Container Device Interface](https://github.com/cncf-tags/container-device-interface) (CDI) notation.
+
+    lxc config device add <instance_name> <device_name> gpu gputype=physical id=<fq_CDI_name>
+
 See {ref}`instances-configure-devices` for more information.
 
+#### Passing an NVIDIA iGPU to a container
+
+If you have NVIDIA runtime libraries and configuration installed on your host and you want to pass these files to your container, you should add a device with the CDI notation.
+Consider the following scenario:
+
+Your host is an NVIDIA single board computer that has a Tegra SoC with an iGPU, and you have an SDK for handling AI workloads installed on the host. You want to create a LXD container and run an inference job inside the container using the iGPU as a backend. This job should run inside a Docker container (or another OCI-compliant runtime).
+To achieve this, complete the following steps:
+
+1. Initialize a LXD container:
+
+       lxc init ubuntu:24.04 t1 --config security.nesting=true --config security.privileged=true
+
+1. Add an iGPU device to your container:
+
+       lxc config device add t1 igpu0 gpu gputype=physical id=nvidia.com/igpu=0
+
+1. Create a `cloud-init` script that installs the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit), and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
```

Comment:
This all feels too specific for the device reference section - it should cover the meanings and valid values of each config key, and give a simple short example of how to pass a device through.

But this looks to me more like a specific how to or tutorial.

What do you think @ru-fu ?

Reply:
It depends on how generally relevant this example is. 
If it is a common use case, moving it to a how-to makes sense.

However, I have the feeling this is very specific, and in this case, turning it into a how-to makes it very visible and presents it as a common recommendation. I can't judge if that is a problem or not.

In general, we have the configuration examples sections here to show how to achieve specific goals, so I think it's okay to leave it here (if we don't want to promote it too much). But of course it is quite long.

Reply:
fixed